airflow/serialization/serializers/timezone.py

```diff
@@ -69,4 +82,28 @@ def deserialize(classname: str, version: int, data: object) -> Timezone:
     if isinstance(data, int):
         return fixed_timezone(data)
 
+    if "zoneinfo.ZoneInfo" in classname:  # capturing backports and stdlib
+        if PY39:
+            from zoneinfo import ZoneInfo
+        else:
+            from backports.zoneinfo import ZoneInfo
+        return ZoneInfo(data)
+
     return timezone(data)
+
+
+def _get_tzinfo_name(tzinfo: datetime.tzinfo | None) -> str | None:
+    if tzinfo is None:
```

Comment:
Should we mentioned that this part vendored from pendulum?

Reply:
It's ported (code is changed). Happy to state that

Reply:
Depends on the license I need to check

Reply:
I think pendulum use MIT licence

Reply:
The license is already included within Airflow, no need to add it separately then. But ill add a notice